vcap_utils/setup.py

```diff
@@ -1,18 +1,23 @@
 #!/usr/bin/env python3
 import os
+from pathlib import Path
 
 from setuptools import setup, find_namespace_packages
 
+# Get package version/metadata
```

Comment:
I looked at how Requests, Falcon, Poetry, and other repos did this. It looks like there isn't a super clear way to do this, so this was the cleanest in my opinion. Open to easier ways.